sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -394,16 +394,25 @@ def set_configuration_setting(
         except binascii.Error:
             raise binascii.Error("Connection string secret has incorrect padding")
 
-    @distributed_trace
-    def delete_configuration_setting(self, key, label=None, **kwargs):
-        # type: (str, Optional[str], **Any) -> ConfigurationSetting
+    @overload
+    def delete_configuration_setting(self, key, **kwargs):
```

Comment:
This will make "label" a keyword only argument which is breaking.

Reply:
Fixed